### ISSUE TITLE
E for emotes

### ISSUE THREAD

**Issue by maddawg836** [SebastianRask/Pocket-Plays-for-Twitch#6]
I just installed pocket plays on my me tablet and almost all the twitch emotes I see are big E. Doesn't seem to happen in my other 2 devices, but I haven't updated those yet.

**Comment by SebastianRask** [comment]
The 'E' emoji is shown when the application fails to download the emote from the Twitch/BTTV API. 
I'm guessing this error emoji is incorrectly being cached in place of the correct emoji.

You should be able to resolve this by going to Settings -> Applications -> Pocket Plays and tapping clear data. This will reset the app.

**Comment by maddawg836** [comment]
Thanks for the help!

I've reinstalled a few times before but maybe it carried over some data. Now tried force stopped, clear cache and data a couple times. The last time after a refresh, I disabled save emotes in local storage and still no luck :(
I just got a T280NZKAXAR (Galaxy Tab A 7") on 5.1.1. Hard to watch twitch without your great app!

**Comment by SebastianRask** [comment]
That sounds very strange. Thanks for reaching out! I'll take a look at it when I find the time.

**Comment by maddawg836** [comment]
I think I may have gotten this to work. I saw a few emotes shown for twitch but not all. I kept switching the option to save locally off then loaded a stream and that gave me more emotes. Then switched to save them and they stayed. cleared the memory and redid that and they are all there now! I will check again later and restart the device. Hopefully it's still there. I'll close this if everything works!
